### PR TITLE
Add Nginx $args issue to Common Problems page

### PR DESCRIPTION
**Pull request by kadamwhite**
Originating in https://github.com/WP-API/WP-API/issues/1926 and possibly https://github.com/WP-API/WP-API/issues/2647

### CODE REVIEW

File: guide/problems/index.md [hunk 1]
```diff
@@ -3,6 +3,27 @@ title: Common Problems
...
+
+
+Query Parameters are ignored
+----------------------------
+
+If you find that you cannot use `?filter[]=`, `?page=` or any other query parameters, your server may not be properly configured to detect them. If you are using Nginx to serve your website, look for a `try_files` line in your site configuration. If it looks like this:
+
+~~~
+try_files $uri $uri/ /index.php$args;
+~~~
```
Comment (rmccue) [inline guide/problems/index.md]: These should be backticks instead to match the rest of the site.

File: guide/problems/index.md [hunk 2]
```diff
@@ -3,6 +3,27 @@ title: Common Problems
 ---
 We try and make the REST API as easy as possible, but APIs are hard. Here's a collection of common issues we run into.
 
+* TOC
+{:toc}
```
Comment (rmccue) [inline guide/problems/index.md:7]: Is this a thing? Never seen this before.

Comment (kadamwhite) [reply guide/problems/index.md:7]: This is indeed a thing, it is one of my favorite things about Jekyll / GH pages actually:

image

Updated the backticks!

Comment (BE-Webdesign) [reply guide/problems/index.md:7]: Table of contents ftw!

**Pull request merged**